### PR TITLE
Add Config utility classes

### PR DESCRIPTION
**Pull request by phlax**
Wraps pootle_config and allows Site, Model, and Object config to be represented as dictionaries, eg

```python
>>> from pootle_config.utils import SiteConfig, ModelConfig, ObjectConfig
>>> site_conf = SiteConfig()
>>> site_conf["foo"] = "bar"
>>> site_conf["foo"]
"bar"
>>> SiteConfig()["foo"]
"bar"
>>> from pootle_project.models import Project
>>> model_conf = ModelConfig(Project)
>>> model_conf["foo"] = "bar2"
>>> ModelConfig(Project)["foo"]
"bar2"
>>> from pootle_project.models import Project
>>> object_conf = ObjectConfig(Project.objects.first())
>>> object_conf["foo"] = "bar3"
>>> ObjectConfig(Project.objects.first())["foo"]
"bar3"
```

### CODE REVIEW

File: pootle/apps/pootle_config/managers.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def list_config(self, key=None, model=None):
                 conf = conf.filter(key__in=key)
         elif key is not None:
             conf = conf.filter(key=key)
-        for item in conf:
+        for item in conf.order_by("key", "pk"):
```
Comment (ta2-1) [inline pootle/apps/pootle_config/managers.py]: why is this necessary?

Comment (phlax) [inline pootle/apps/pootle_config/managers.py]: because there may be multiple k/v pairs with same k, this ensures it gets the last - which is at least predictable

Comment (phlax) [inline pootle/apps/pootle_config/managers.py]: but i certainly thought about making this configurable - because most times you dont care

**Comment by ta2-1** [comment]
lgtm
